### ISSUE TITLE
Dual 4/5 release-info contains listing for 0.1 which does not exist for CHICKEN 5

### ISSUE THREAD

**Issue by sjamaan** [ursetto/feature-test-egg#2]
Hi Jim,

I noticed vandusen complaining as follows:

```
[eggs] feature-test version 0.1 (for CHICKEN 5) could not be downloaded: Could not download files-list  "http://code.call-cc.org/files-list?egg=feature-test;egg-release=0.1;chicken-release=5" -- Client error: 404 Release "0.1" for egg "feature-test" (CHICKEN version 5) doesn't exist
```

It looks like the problem here is that you're using the same release-info file for both CHICKEN 4 and CHICKEN 5, and it lists `http://code.call-cc.org/files-list?egg={egg-name};egg-release={egg-release};chicken-release={chicken-release}` as the location for the files list. However, for `chicken-release=5` that is unable to find the egg, as there is no `release/5/feature-test/tags/0.1` directory.

You could solve this by splitting up the release-info into 4 and 5, or just copying the tag in SVN to `release/5`. The egg might not work but at least this will prevent the error.

**Comment by ursetto** [comment]
Is this recurring? That would be annoying. I'm aware of this issue but it's harmless as long as it doesn't spam the channel. This same problem will affect several eggs of mine.

I guess the split release-info option would work. Alternatively I did create corresponding git tags for nearly all the old SVN tags, so I could point to git for old versions as well. It's just that I did not test pulling old versions from git, so I left SVN in place for safety.

Let me think about this.

**Comment by sjamaan** [comment]
yeah, it's really spamming the channel; whenever henrietta-cache sees an error it is echoed to the channel (so, it happens every hour).

**Comment by ursetto** [comment]
Crap. I will work on this then.

**Comment by ursetto** [comment]
I am hoping I fixed the spam now in 658c59fd34da1b461aa208c6ff54c0bb1bac7257.
